services/ui-src/src/styles/index.scss

```diff
@@ -174,3 +174,31 @@ button {
 .optionalText {
   font-weight: normal;
 }
+
+// Tech Specs Link Styles
+.tech-spec-link {
```

Comment:
@ailZhou might I poke your brain here to see why the default colors for links aren't applying to this link?

Megan mentioned that she's not seeing these colors update as such:
Not yet click MDCT/primary (Hex: #0071BC)
Hover: MDCT/primary-darker (Hex: #004F84)
Once clicked, the color should be - MDCT/visited (Hex: #4C2C92)